src/dev/build/tasks/siem_cycle_task.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const madge = require('madge');
+
+export const SiemCycleTask = {
+  description: 'Checking SIEM project for circular dependencies',
+
+  async run(config, log, build) {
+    const buildRoot = build.resolvePath();
+    const siemFolder = `${buildRoot}/x-pack/plugins/siem`;
```

Comment:
```suggestion
    const siemFolder = build.resolvePath('x-pack/plugins/siem');
```

Reply:
Changed the build around per conversations to use typescript and not hook directly into transpiled Javascript. So, this comment is outdated and I will mark it resolved.